template_config.yml

```diff
@@ -36,13 +37,12 @@ pulp_settings:
   - /tmp
   allowed_import_paths:
   - /tmp
-pulpcore_branch: master
-pulpcore_pip_version_specifier: null
+pulpcore_branch: '3.11'
+pulpcore_pip_version_specifier: '~3.11.0'
```

Comment:
I haven't seen it being used without equal sign https://www.python.org/dev/peps/pep-0440/#compatible-release
But maybe I'm just not aware of this use.